Dockerfiles/midstream/run_tests.py

```diff
@@ -35,7 +35,7 @@ def run_extract_operator_bundle(self):
         return result
 
     def run_validate_operator_bundle(self):
-        exec_cmd = "ansible-playbook -vvv -i localhost, --connection local \
+        exec_cmd = "ansible-playbook -vvvv -i localhost, --connection local \
```

Comment:
-1

Verbosity should be configurable.
And midstream expect configuration just with using env variables.

Reply:
I agree with it. Will add to the implementation.